sfputil/main.py

```diff
@@ -1306,6 +1306,61 @@ def reset(port_name):
 
         i += 1
 
+
+# 'power' subgroup
+@cli.group()
+def power():
+    """Enable or disable power of SFP transceiver"""
+    pass
+
+
+# Helper method for setting low-power mode
+def set_power(port_name, enable):
+    physical_port = logical_port_to_physical_port_index(port_name)
+    sfp = platform_chassis.get_sfp(physical_port)
+
+    if is_port_type_rj45(port_name):
+        click.echo("Power disable/enable is not available for RJ45 port {}.".format(port_name))
+        sys.exit(EXIT_FAIL)
+
+    try:
+        presence = sfp.get_presence()
+    except NotImplementedError:
+        click.echo("sfp get_presence() NOT implemented!")
+        sys.exit(EXIT_FAIL)
+
+    if not presence:
+        click.echo("{}: SFP EEPROM not detected\n".format(port_name))
+        sys.exit(EXIT_FAIL)
+
+    try:
+        result = platform_chassis.get_sfp(physical_port).set_power(enable)
+    except (NotImplementedError, AttributeError):
+        click.echo("This functionality is currently not implemented for this platform")
+        sys.exit(ERROR_NOT_IMPLEMENTED)
+
+    if result:
+        click.echo("OK")
+    else:
+        click.echo("Failed")
```

Comment:
@mihirpat1 : Since this only api do we need to address this comment?

Reply:
@AnoopKamath - Yes - returning non-zero in case of an error will help the caller in knowing if the command was successful or not so that the caller can handle error scenario accordingly.

Reply:
@mihirpat1 - The platform handler, which toggles GPIO pin sends TRUE on success. If we need a non-zero in failed case, we should come up with different ERROR codes?

Reply:
@AnoopKamath Based on my understanding, the set_power API returns False upon [error](https://github.com/AnoopKamath/sonic-platform-common/blob/92a218c7974e8c6453c3ed19a3f984c323ca5d2e/sonic_platform_base/sfp_base.py#L396).
The `result` variable in the current context already has this info.
The current API can return `result` and the caller can return the error code to the user. You can return `EXIT_FAIL` to the user as well upon error and `EXIT_SUCCESS` upon success.